site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -183,21 +183,38 @@ please [file an issue](https://github.com/googleforgames/agones/issues).
 #### SetLabel(key, value)
 
 This will set a [Label](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) value on the backing `GameServer`
-record that is stored in Kubernetes. To maintain isolation, the `key` value is automatically prefixed with "agones.dev/sdk-".
+record that is stored in Kubernetes. 
+
+To maintain isolation, the `key` value is automatically prefixed with value **"agones.dev/sdk-"**. This is done for two 
+main reasons:
+*  The prefix allows the developer to always know if they are accessing or reading a value that could have come, or 
+   may be changed by the client SDK. Much like `private` vs `public` scope in a programming language - The Agones SDK only 
+   gives you access to part of the set of labels and annotations that exist on a GameServer.
```

Comment:
nit: gives you access to write to part of the set of labels...

Reply:
agree with this clearly point out that you can edit only the labels that have the prefix.